packages/roosterjs-editor-dom/lib/utils/findClosestElementTypeAncestor.ts

```diff
@@ -0,0 +1,25 @@
+import getTagOfNode from './getTagOfNode';
+import { NodeType } from 'roosterjs-editor-types';
+
+/**
+ * Find closest element ancestor of a specific tag element
+ * @param node Find ancestor start from this node
+ * @param root Root node where the search should stop at. The return value can never be this node
+ * @param tag Tag of the element to find.
+ * @returns An HTML element which is the closest ancestor with the provided Tag, returns null if not found and reached the root element
+ */
+export default function findClosestElementTypeAncestor(node: Node, root: Node, tag: string): Node {
```

Comment:
this file is almost the same with findClosestElementAncestor, and findClosestElementAncestor can already cover this file. the selector parameter already work for tag name

Reply:
Removing this.